src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -129,13 +152,64 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
   }, [state]);
 
   const onSubmit = useCallback(
-    async (vals: Partial<FormValues>) => {
-      if (!user) return;
+    async (
+      vals: Partial<FormValues>,
+      userStatuses: UserStatus[],
+      showUserStatuses: boolean
+    ) => {
+      if (!user || formError) return;
       try {
         // unfortunately the typing is off for react-hook-forms.
-        if (!!venueId)
-          await updateVenue({ ...(vals as VenueInput), id: venueId }, user);
-        else await createVenue(vals as VenueInput, user);
+        if (!!venueId) {
+          await updateVenue(
+            {
+              ...(vals as VenueInput),
+              id: venueId,
+              userStatuses,
+              showUserStatus: showUserStatuses,
+            },
+            user
+          );
+
+          console.log(sovereignVenueId);
+
+          if (sovereignVenueId)
+            await updateVenue(
+              {
+                ...(vals as VenueInput),
+                id: sovereignVenueId,
+                parentId: sovereignVenue?.parentId,
+                userStatuses,
+                showUserStatus: showUserStatuses,
+              },
+              user
+            ).then(() => {
+              console.log(sovereignVenue);
+
+              if (sovereignVenue) {
+                console.log(
+                  "updating redux sov venue",
+                  showUserStatuses,
+                  userStatuses
+                );
```

Comment:
```suggestion
```

Reply:
Removed in: https://github.com/sparkletown/sparkle/pull/1532/commits/0e1ea16c81870c898660c5e5b60aa9a143b57c80